__tests__/data/airship_input.json

```diff
@@ -89,28 +102,37 @@
         },
         "screen": {
           "density": 2
-        },
-        "page": {
-          "path": "/destinations/airship",
-          "referrer": "",
-          "search": "",
-          "title": "",
-          "url": "https://docs.rudderstack.com/destinations/airship",
-          "category": "destination",
-          "initial_referrer": "https://docs.rudderstack.com",
-          "initial_referring_domain": "docs.rudderstack.com"
         }
       },
-      "type": "identify",
+      "type": "track",
       "messageId": "84e26acc-56a5-4835-8233-591137fca468",
       "session_id": "3049dc4c-5a95-4ccd-a3e7-d74a7e411f22",
       "originalTimestamp": "2019-10-14T09:03:17.562Z",
       "anonymousId": "123456",
-      "event": "product clicked",
       "userId": "testuserId1",
       "properties": {
```

Comment:
Can we have a test case with empty properties?

Reply:
Added.